apps/libc/dynamic/qsort.c

```diff
@@ -284,7 +284,7 @@ Return Value:
         //
 
         if (SmallerIndex != LargerIndex) {
-            QUICKSORT_SWAP(ArrayBase, ElementSize, SmallerIndex, LargerIndex);
+            QUICKSORT_SWAP((char *)ArrayBase, ElementSize, SmallerIndex, LargerIndex);
```

Comment:
In this case it looks like the use of our new POINTER_ADD macro would need to be inside QUICKSORT_ELEMENT